integration-tests/checkstyle-init.patch

```diff
@@ -7,7 +7,7 @@
 +    <dependency>
 +      <groupId>org.assertj</groupId>
 +      <artifactId>assertj-core</artifactId>
-+      <version>${assertj.version}</version>
++      <version>3.26.3<!-- Renovate: org.assertj:assertj-bom --></version>
```

Comment:
I opted for this syntax, as it's a single-line way (thus compatible with Renovate's regex manager) to link versions to their dependencies, without splitting this logic across files.